Assets/Scripts/Furnitures/Shootable/WanderAI.cs

```diff
@@ -41,10 +44,19 @@ private void Start()
 
 		// Populate the speed from the stats
 		agent.speed = info.speed;
+
+		lurePos = Vector3.zero;
 	}
 
 	private void Update()
 	{
+		lure = GameObject.FindWithTag("Lure");
```

Comment:
`FindWithTag` is  very expensive and should be avoided in `Update` where possible. Consider using a `Physics.OverlapSphere` to search for lures within a range instead